server/components/buttons/middleware.js

```diff
@@ -168,10 +169,35 @@ export function getButtonMiddleware({
                 brandedDefault: experiments.isFundingSourceBranded
             };
 
+            const cplCompPayload = {
+                'second-render-middleware': {
+                    start: middlewareStartTime,
+                    tt:    Date.now() - middlewareStartTime
+                }
+            };
+
+            logger.info(req, 'CPL_LATENCY_METRICS_SECOND_RENDER_MIDDLEWARE');
+            logger.track(req, {
+                [FPTI_KEY.STATE]:                 'CPL_LATENCY_METRICS',
+                [FPTI_KEY.TRANSITION]:            'process_server_metrics',
+                [FPTI_KEY.PAGE]:                  'main:xo:paypal-components:smart-payment-buttons',
+                [FPTI_KEY.CONTEXT_ID]:            buttonSessionID,
+                [FPTI_KEY.CPL_COMP_METRICS]:      cplCompPayload,
+                [FPTI_KEY.CPL_QUERY_METRICS]:     {},
+                [FPTI_KEY.CPL_CHUNK_METRICS]:     {}
+            }, {});
+            const responseStartTime = Date.now();
+
             const pageHTML = `
                 <!DOCTYPE html>
-                <head></head>
-                <body data-nonce="${ cspNonce }" data-client-version="${ client.version }" data-render-version="${ render.version }">
+                <head>
+                    <script type="text/javascript" nonce="${ cspNonce }">
```

Comment:
With html5 doctype this is the default behavior of script so no need to specify it
```suggestion
                    <script nonce="${ cspNonce }">
```